addons/portal/tests/test_login.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import HttpCase, new_test_user, tagged
+
+
+@tagged('post_install')
+class TestWebLogin(HttpCase):
+    def test_web_login_public_user(self):
```

Comment:
same here (portal user)

I think you can create only 1 user in `setUp` and use it in both tests, what do you think ?